package.nls.json

```diff
@@ -8,6 +8,7 @@
    "makefile-tools.command.makefile.setBuildConfiguration.title": "Makefile: Set the current build configuration",
    "makefile-tools.command.makefile.setBuildTarget.title": "Makefile: Set the target to be built by make",
    "makefile-tools.command.makefile.setLaunchConfiguration.title": "Makefile: Set the make launch configuration",
+   "workbench.action.openWorkspaceSettings.title": "Preferences: Open Workspace Settings",

```

Comment:
Not sure how to properly localize this string... it's different than all the rest, it doesn't belong to makefile tools, but how do I reference a string localize-able from another extension or from vscode itself?

Reply:
Good question. I'm not sure I know of a way to do this. 

Would it be bad to simply localize it for ourselves here? Presumably the strings will be the same.